docker/Dockerfile.ci

```
@@ -58,8 +58,9 @@ RUN groupadd -g 1500 rust \
   && echo 'rust ALL=NOPASSWD: ALL' >> /etc/sudoers.d/10-rust \
   && echo 'Defaults    env_keep += "DEBIAN_FRONTEND"' >> /etc/sudoers.d/env_keep
 
-# Ensure we are ready with cargo hakari
+# Ensure we are ready with cargo hakari and cargo deny
 RUN cargo install cargo-hakari
+RUN cargo install cargo-deny
 RUN chown -R rust:rust /usr/local/cargo
```

```suggestion
RUN cargo install cargo-hakari && \
    cargo install cargo-deny && \
    chown -R rust:rust /usr/local/cargo
```

There's no need to have dedicated image layers for this, it just adds overhead when pushing and pulling the image.